### ISSUE TITLE
If statements are stripped when flushed.

### ISSUE THREAD

**Issue by hackeryarn** [tmont/nginx-conf#15]
If you load a config with an `if` statement inside it, and then call flush on it, the `if` statement brackets get stripped and it is not handle properly.

to reproduce:
```javascript
const NginxConfFile = require('nginx-conf').NginxConfFile;

NginxConfFile.create('nginx.conf', (err, conf) => {
    conf.flush();
 );
```

**Comment by Volem** [comment]
This looks like an enhancement since I don't see a code block to specifically handle if statements.

**Comment by tmont** [comment]
That sounds bad. I'll take a look at it later today.

**Comment by tmont** [comment]
@hackeryarn Can you provide an example configuration that fails? I just added a few tests in d36f18e to more robustly test `if` statements and was unable to reproduce what you described.

**Comment by tmont** [comment]
Thanks for the bug report and the failing test. This has been fixed in NPM at  `nginx-conf@1.4.0`.

**Comment by hackeryarn** [comment]
@tmont thank you so much for the quick fix! I agree this was a weird scenario but it's great that we have it covered now.
